packages/shared_preferences/shared_preferences/README.md

```diff
@@ -85,6 +85,31 @@ If this is problematic for your use case, you can thumbs up
 interest in APIs that provide direct (asynchronous) access to the underlying
 preference store, and/or subscribe to it for updates.
 
+### Migration and Prefixes
+
+By default, the `SharedPreferences` plugin will only read (and write) preferences
+that begin with the prefix `flutter.`. This is all handled internally by the plugin
+and does not require manually adding this prefix.
+
+Alternatively, `SharedPreferences` can be configured to use any prefix by adding 
+a call to `setPrefix` before any instances of `SharedPreferences` are instantiated.
+Calling `setPrefix` after an instance of `SharedPreferences` is  created will fail.
+
+Setting the prefix to an empty string `''` will allow access to all preferences created
+by any non-flutter versions of the app (for migrating from a native app to flutter).
+
+If the prefix is set to a value such as `''` that causes it to read values that were 
```

Comment:
Very minor nit: I would make this sentence part of the previous paragraph rather than its own paragraph.